test/configs/zombieDancelightUpgrade.json

```diff
@@ -0,0 +1,66 @@
+{
+    "settings": {
+        "timeout": 1000,
+        "provider": "native"
+    },
+    "relaychain": {
+        "chain_spec_path": "specs/tanssi-relay.json",
+        "default_command": "../target/release/tanssi-relay",
+        "default_args": ["--no-hardware-benchmarks", "-lparachain=debug", "--database=paritydb", "--no-beefy"],
+        "genesis": {
+            "runtimeGenesis": {
+                "patch": {
+                    "configuration": {
+                        "config": {
+                            "async_backing_params": {
+                                "allowed_ancestry_len": 2,
+                                "max_candidate_depth": 3
+                            },
+                            "scheduler_params": {
+                                "scheduling_lookahead": 2,
+                                "num_cores": 4
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "nodes": [
+            {
+                "name": "alice",
+                "ws_port": "9947",
+                "validator": true
+            },
+            {
+                "name": "bob",
+                "validator": true
+            }
+        ]
+    },
+    "parachains": [
```

Comment:
this is needed because moonwall does not accept zombienet configs without parachains

Reply:
Would be nice to have a comment in the file, but this is json

Reply:
I will make sure @timbrinded is aware once he comes back